gravitee-apim-console-webui/src/components/gio-ee-unlock-dialog/gio-ee-unlock-dialog.component.html

```diff
@@ -23,25 +23,11 @@
 </div>
 
 <mat-dialog-content class="gio-ee-unlock-dialog__content">
-  <img
-    class="gio-ee-unlock-dialog__content__image"
-    [src]="featureMoreInformation.image"
-    [alt]="featureMoreInformation.title || 'Unlock Gravitee Enterprise'"
-  />
-  <h2
-    class="gio-ee-unlock-dialog__content__title"
-    [innerHTML]="featureMoreInformation.title || 'Unlock Gravitee Enterprise' | safe : 'html'"
-  ></h2>
-  <div class="gio-ee-unlock-dialog__content__description" [innerHTML]="featureMoreInformation.description | safe : 'html'"></div>
+  <img class="gio-ee-unlock-dialog__content__image" [src]="featureInfo.image" [alt]="featureInfo.title || 'Unlock Gravitee Enterprise'" />
+  <h2 class="gio-ee-unlock-dialog__content__title" [innerHTML]="featureInfo.title || 'Unlock Gravitee Enterprise' | safe : 'html'"></h2>
```

Comment:
Not sure how this will be interpreted but in the doubt I would go for 
`(featureInfo.title || 'Unlock Gravitee Enterprise') | safe : 'html'`